src/linagora.esn.unifiedinbox/i18n/fr.json

```diff
@@ -76,6 +77,7 @@
   "File %s ignored as its size exceeds the %s limit": "Le fichier %s ignore que sa taille dépasse la limite %s",
   "Filter name": "Nom du filtre",
   "Filters": "Filtres",
+  "Filters set": "Filtres créés",
```

Comment:
`Error setting filters` and `Filters set` translations do not have the same meaning.
for one `set` means created and for the other means `configured`.

so either we use
```json
 "Filters set": "Filtres configured"
```
 or 
```json
"Error setting filters": "Erreur lors de la creation des filtres"
```

or something else completely different, something that better explains what `set` means.